tests/test_2678_same_backend.py

```diff
@@ -0,0 +1,19 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import numpy as np  # noqa: F401
+import pytest  # noqa: F401
+
+import awkward as ak
+
+
+def test_where():
+    result = ak.where(
+        [True, False, False], ak.to_backend([1, 2, 3], "typetracer"), [4, 5, 6]
+    )
+    assert ak.backend(result) == "typetracer"
+
+
+def test_almost_equal():
+    assert not ak.almost_equal(
+        [True, False, False], ak.to_backend([True, False, False], "typetracer")
+    )
```

Comment:
New mixed-backend support for `almost_equal`.

Reply:
:heavy_check_mark: